localisation/00_wc_sad.csv

```diff
@@ -728,4 +750,6 @@ EVTDESC_WCUTF.3_thorim;After centuries spend in lethargy you have finally seen t
 EVTOPTA_WCUTF.3;Forgive me, my Makers.;;;;;;;;;;;;;x
 EVTOPTB_WCUTF.3;Rest in peace, brother.;;;;;;;;;;;;;x
 ulduar_closed_down;Ulduar is Closed Down;;Ulduar ist geschlossen;;;;;;;;;;;x
-ulduar_closed_down_desc;Ulduar has been closed down for unknown reasons.;;Ulduar ist aus unbekannten Gr�nden geschlossen worden.;;;;;;;;;;;x
\ No newline at end of file
+ulduar_closed_down_desc;Ulduar has been closed down for unknown reasons.;;Ulduar ist aus unbekannten Gr�nden geschlossen worden.;;;;;;;;;;;x
+recruit_old_god_guard;Raise the Old God Guard.;;;;;;;;;;;;;x
+recruit_old_god_guard_desc;Old God can create legions of servants from his abberant flesh and release them upon any fools who dared to face him.;;;;;;;;;;;x
```

Comment:
I'm not an expert, but shouldn't it be...
```suggestion
recruit_old_god_guard_desc;The Old Gods can create legions of servants from their aberrant flesh and release them upon any fools who dared to face them.;;;;;;;;;;;x
```
Anyway, this branch needs a review of native Englishman.